surface_potential_analysis/kernel/kernel.py

```diff
@@ -658,3 +667,52 @@ def truncate_noise_kernel(
             "eigenvalue": operators["eigenvalue"][args],
         }
     )
+
+
+def get_coefficient_matrix_taylor(
+    *,
+    true_noise_coeff: np.ndarray[tuple[int, int], np.dtype[np.float64]],
+    delta_k: np.floating[Any],
+    n: int = 1,
+) -> np.ndarray[tuple[int, int], np.dtype[np.float64]]:
+    coeff_to_solve = np.array(
+        [
+            np.array(
+                [
+                    (((-1) ** m) / (factorial(2 * m))) * ((i * delta_k) ** (2 * m))
+                    for i in np.arange(0, n + 1)
+                ],
+            )
+            for m in np.arange(0, n + 1)
+        ],
+    )
+    coeff = np.linalg.solve(coeff_to_solve, true_noise_coeff)
+    coeff = coeff.T
+    return np.concatenate([coeff, coeff[1:]])
+
+
+def get_lorentzian_parameter(
```

Comment:
I think a 'get effective Lorentzian parameter' (similar to gaussian which matches the Taylor expansion of the noise to caldera Leggett) would be more helpful here